src/layouts/index.tsx

```diff
@@ -1,4 +1,4 @@
-import Navigation from "../components/nav";
+import Navigation from "../components/Nav";
```

Comment:
This is because the folder name should have been 'nav', not 'nav'. The 'develop' branch doesn't have this problem.
---
Should merge into the 'develop' branch first?😛